openquake/calculators/scenario_damage.py

```diff
@@ -103,7 +103,7 @@ def scenario_damage(riskinputs, param, monitor):
                     # ddds has shape E', D with E' == len(out.eids)
                     for e, ddd in enumerate(ddds):
                         eid = out.eids[e]
-                        ddic[aid, eid][l] = ddd[1:]
+                        ddic[aid, eid][l] += ddd[1:]
```

Comment:
Shouldn't really affect any calculations, right? For a given [asset, event, loss_type], there should be just one damage array. So `+=` and `=` should have identical results?

Reply:
You are right, Anirudh, there is no bug, I was mislead by another issue (for the ZeroGetters it looks like there are multiple contributions for the same event, but they are all zeros so there is no problem anyway).